kebechet/utils.py

```diff
@@ -177,8 +178,8 @@ def get_issue_by_title(ogr_project: BaseGitProject, title: str):
 
 
 EXCEPTION_ISSUE_BODY = """## Description
-This is an automated issue generated by Kebechet. The {manager} manager threw an exception ({exception_type}) at
-runtime while working on the following repository https://github.com/{slug}
+This is an automated issue generated by Kebechet v{version}. The {manager} manager threw an exception ({exception_type})
```

Comment:
It's imported as `keb_version`, right?
```suggestion
This is an automated issue generated by Kebechet v{keb_version}. The {manager} manager threw an exception ({exception_type})
```

Reply:
I called it below by writing:
```
EXCEPTION_ISSUE_BODY.format(..., version=keb_version,...)
```

Reply:
ah, right, sorry I missed that :+1: